sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -1045,7 +1045,17 @@ static void _handle_nbr_sol(gnrc_netif_t *netif, const ipv6_hdr_t *ipv6,
                     break;
             }
         }
-        reply_aro = _copy_and_handle_aro(netif, ipv6, nbr_sol, aro, sl2ao);
+        if (aro && sl2ao) {
+        /* If no SLLAO is included, then any included ARO is ignored. */
```

Comment:
```suggestion
            assert(reply_aro == NULL);
            /* If no SLLAO is included, then any included ARO is ignored. */
```

Reply:
If that concerns you, I would rather move the declaration and initialization of `reply_aro` down there, keeping it close to where it is used.

Reply:
That's fine as well.